addons/mrp/models/mrp_workorder.py

```diff
@@ -593,7 +593,7 @@ def button_start(self):
                 continue
             # As button_start is automatically called in the new view
             if wo.state in ('done', 'cancel'):
-                continue
+                raise UserError(_('You cannot start a work order that is already done or cancelled'))
```

Comment:
I'd delete the comment about the button being automatically called since it's not valid anymore

also .pot file :wink:

Reply:
Done, Thank you for the review ;)